src/coreclr-debug/activate.ts

```diff
@@ -23,32 +23,39 @@ export async function activate(thisExtension : vscode.Extension<any>, context: v
     _logger = logger;
 
     if (!CoreClrDebugUtil.existsSync(_debugUtil.debugAdapterDir())) {
```

Comment:
I think you should probably just delete this check all together and move both checks into `completeDebuggerInstall`.